src/EditorFeatures/Core.Wpf/BackgroundWorkIndicator/WpfBackgroundWorkIndicatorFactory.cs

```diff
@@ -48,22 +45,32 @@ internal sealed partial class WpfBackgroundWorkIndicatorFactory : IBackgroundWor
             _threadingContext.ThrowIfNotOnUIThread();
 
             // If we have an outstanding context in flight, cancel it and create a new one to show the user.
-            _currentContext?.CancelAndDispose();
+            _currentContext?.Dispose();
```

Comment:
The platform supports multiple, so we can make the decision if we want multiple to be shown